Pod/Classes/BONText.h

```diff
@@ -84,6 +84,11 @@ typedef NS_ENUM(NSUInteger, BONFigureSpacing) {
 @property (nonatomic) NSUnderlineStyle strikethroughStyle;
 @property (strong, nonatomic, BONNullable) UIColor *strikethroughColor;
 
+/**
+ *  Assign an array of @p BONTags to use in styling substrings.
```

Comment:
#pp `@p BONTag objects` so you end up with "`BONTag` objects`, and not "`BONTags`", in the formatted documentation.